packages/nimble-components/src/spinner/index.ts

```diff
@@ -12,7 +14,15 @@ declare global {
  * A Nimble-styled spinner component.
  * A spinner is an animating indicator that can be placed in a particular region of a page to represent loading progress, or an ongoing operation, of an indeterminate / unknown duration.
  */
-export class Spinner extends FoundationElement {}
+export class Spinner extends FoundationElement {
+    /**
+     * @public
+     * @description
+     * Type of spinner
```

Comment:
nit: not much better, but it matches the docs from other components' appearance property
```suggestion
     * The appearance the spinner area should have.
```